src/taco-remote-lib/ios/iosAppRunnerHelper.ts

```diff
@@ -22,24 +22,26 @@ import utils = require ("taco-utils");
 
 import UtilHelper = utils.UtilHelper;
 
-var proxyInstance: child_process.ChildProcess = null;
 var promiseExec: (...args: any[]) => Q.Promise<any> = Q.denodeify(UtilHelper.loggedExec);
 
 class IosAppRunnerHelper {
     public static startDebugProxy(proxyPort: number): Q.Promise<child_process.ChildProcess> {
-        if (proxyInstance) {
-            proxyInstance.kill("SIGHUP"); // idevicedebugserver does not exit from SIGTERM
-            proxyInstance = null;
+        if (!global.tacoRemoteLib) {
+            global.tacoRemoteLib = {};
```

Comment:
This is lazily initializing the global.tacoRemoteLib object / namespace. If this is the first time we've ever tried to look at the tacoRemoteLib global storage, it won't exist, so we create it now. If we've run this method before (or another instance of this package has, e.g. taco-remote-lib@1.2.1) then the tacoRemoteLib global object will have been initialized already and we can proceed directly to using it.